core/chains/evm/log/mocks/orm.go

```diff
@@ -15,20 +15,20 @@ type ORM struct {
 	mock.Mock
 }
 
-// CreateBroadcast provides a mock function with given fields: blockHash, blockNumber, txIndex, logIndex, jobID, qopts
-func (_m *ORM) CreateBroadcast(blockHash common.Hash, blockNumber uint64, txIndex uint, logIndex uint, jobID int32, qopts ...pg.QOpt) error {
```

Comment:
As no mocks of the orm are used this change is not breaking

Reply:
Why are we generating a mock that we don't use? IMHO it should be removed.

Reply:
I'm completely fine with that, I'm just the messenger that ran `go generate` :) I feel like we might have some other mocks that can be removed besides these ones. Worth a ticket to check them all?

Reply:
Yeah, if you think there are some more out there, go for it.